fuse_examples/tests/test_notebook_hello_world.py

```diff
@@ -0,0 +1,16 @@
+import os
+import unittest
+from testbook import testbook
+
+class NotebookHelloWorldTestCase(unittest.TestCase):
+    
+    def test_example_2(self):
+        # Execute the whole notebook and save it as an object
+        with testbook('fuse_examples/tutorials/hello_world/hello_world.ipynb', execute=True, timeout=600) as tb:
```

Comment:
Isn't the first cell install fuse-med-ml? Should we skip it?

Reply:
It's not something that we want to test? Because it's already been testing in the Jenkins pipeline itself?

I think it won't be an issue to skip it tho.